modules/facilities_api/app/controllers/facilities_api/v1/va_controller.rb

```diff
@@ -11,6 +11,12 @@ class V1::VAController < ApplicationController
     def index
       api_results = api.get_facilities(lighthouse_params)
 
+      if Flipper.enabled?(:facilities_locator_mobile_covid_online_scheduling) && covid_mobile_params?
+        api_results.each do |api_result|
+          api_result.tmp_covid_online_scheduling = mobile_api_get_by_id(api_result.id)
```

Comment:
Does `mobile_api_get_by_id` return a boolean? Do we need to cast it or something?